script/deploy-canary-demo

```diff
@@ -9,7 +9,11 @@ cd $GITHUB_WORKSPACE/demos/serverless
 npm ci
 
 echo "Deploying to alpha stage for canary"
-npm run deploy -- -b chime-sdk-demo-canary -s chime-sdk-demo-canary -o chime-sdk-demo-canary -i eu-south-1 -t -l
+npm run deploy -- -b chime-sdk-demo-canary -s chime-sdk-demo-canary -o chime-sdk-demo-canary -u false -i eu-south-1 -t -l
```

Comment:
This is only for standup and not canary so i think you can skip this.

Reply:
but if i dont update it the prod the standup link will use Chime Meetings SDK